src/databricks/labs/ucx/install.py

```diff
@@ -1037,6 +1040,34 @@ def validate_and_run(self, step: str):
         if not self.validate_step(step):
             self.run_workflow(step)
 
+    @staticmethod
+    def enable_files_in_repos():
+        client = ApiClient()
+
+        # check if "enableProjectTypeInWorkspace" and "enableWorkspaceFilesystem" are set to false
+        project_type = client.do("GET", "/api/2.0/workspace-conf", {"keys": "enableProjectTypeInWorkspace"})
+        workspace_file_system = client.do("GET", "/api/2.0/workspace-conf", {"keys": "enableWorkspaceFilesystem"})
+
+        logger.debug("Checking Files In Repos configuration")
+
+        if project_type["enableProjectTypeInWorkspace"] == "false":
+            logger.debug("enableProjectTypeInWorkspace is False, enabling the config")
+            client.do(
+                "PATCH",
+                "/api/2.0/workspace-conf",
```

Comment:
Why ws.workspace_conf.set_status() doesn't work? Did you try that?

Reply:
Blocked by https://github.com/databricks/databricks-sdk-py/issues/500